Gopkg.toml

```diff
@@ -109,3 +109,7 @@
 [[constraint]]
   name = "go.uber.org/automaxprocs"
   version = "=1.2.0"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Why to take master, Master is always changing. This might break functionality.
You can take any stable release to avoid functionality break.

Reply:
Thanks Himanshu for review, I have fixed this. Thanks

Reply:
Thanks